curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-bracket-notation-to-find-the-last-character-in-a-string.english.md

```diff
@@ -38,15 +38,11 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
-var firstName = "Ada";
-var lastLetterOfFirstName = firstName[firstName.length - 1];
```

Comment:
Again, the hint (on line 18, this time) stops making sense without this.

Reply:
Good catch. I will try to move the example into the description section.

Reply:
I added an example based on the challenge description.